srcpkgs/worker/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'worker'
+pkgname=worker
+version=4.12.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXinerama-devel libXft-devel file-devel lua54-devel
+	$(vopt_if udisks dbus-devel)"
+depends=$(vopt_if udisks udisks2)
```

Comment:
I don't think UDisks support makes it a dependency. When Worker first starts and generates the configuration, it shows this dialog:

![image](https://github.com/void-linux/void-packages/assets/304760/8d2a4c93-e33b-49d9-b33b-ffc6b10c0684)

On my machine with a system bus but no UDisks service, there are no error messages, console or otherwise. Merely the "Volume manager" dialog is not functional:

![image](https://github.com/void-linux/void-packages/assets/304760/f2ec00bc-8a98-4d58-aebb-8226a492a336)

IMHO, an `INSTALL.msg` that suggests users might want to install `udisks2` if they want to manage volumes from Worker would suffice. FWIW, the [Debian package](https://packages.debian.org/bookworm/worker) doesn't even suggest, let alone recommend or depend on `udisks`.

Reply:
I agree, `udisks2` doesn't need to be in the depends.  I'm not even sure it really warrants an INSTALL.msg.  The dialog, as you mentioned, is helpful, and the worker documentation should be clear enough for users who want to use udisks.